app/services/config_report_status_calculator.rb

```diff
@@ -59,6 +59,10 @@ def pending?
     end
   end
 
+  def to_h
+    Hash[ConfigReport::METRIC.map {|metric| [metric, status_of(metric)] }]
```

Comment:
Isn't this exactly the same result as the `status` method returns?

Reply:
Yup! Very good you passed reviewer mk2 exam test!